HomeWork2/PavelGenitsoyHW2.cpp

```diff
@@ -0,0 +1,204 @@
+#include <iostream>
+
+void digits_in_number() {
+  int num{};
+  std::cout << "\nEnter some number: " << std::endl;
+  std::cin >> num;
+  int sum_digits{}, count{0};
+  if (num > 0) {
+    for (int i{num}; i > 0; i /= 10) {
+      sum_digits += i % 10;
+      ++count;
+    }
+    const auto arithmetic_mean = static_cast<double>(sum_digits) / count;
+    std::cout << "\nThe sum of digits in the number: " << sum_digits
+              << ", and arithmetic mean of digits in the number: "
+              << arithmetic_mean << "\n\n"
+              << std::endl;
+    return;
+  }
+  if (num == 0) {
+    std::cout << "\nThe sum and arithmetic mean of digits in the number: "
+              << num << "\n\n"
+              << std::endl;
+    return;
+  }
+  if (num < 0) {
+    std::cout << "\nTry a number which >0"
+              << "\n\n"
+              << std::endl;
+    return;
+  }
```

Comment:
```suggestion
std::cout << "\nTry a number which >0"
              << "\n\n"
              << std::endl;
```
т.е. вам тут `if` не нужен, у вас это будет дефолтная ветка которая будет исполняться во всех остальных случаях